weave-cloth.lic

```diff
@@ -3,7 +3,7 @@
 =end
 
 DRCT.walk_to(19_065)
-DRCT.walk_to(19_063) unless DRRoom.pcs.empty?
+# DRCT.walk_to(19_063) unless DRRoom.pcs.empty?
```

Comment:
This is commented instead of deleted as I do not do outfitting, and this is possibly needed in the future (with a new room number)? Tell me if i should delete it.